src/content/docs/browser/browser-monitoring/browser-pro-features/browser-logs/get-started.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Getting started
+tags:
+translate:
+metaDescription: 'Learn how to enable and manage Browser Logs in New Relic, including steps for automatic log collection and how to disable the feature.'
+redirects:
+freshnessValidatedDate: never
+---
+
+Enhance your application's observability and debugging capabilities by following these simple setup instructions.
+
+### Prerequisites
+
+* A **Pro** or **Pro + SPA** browser agent
+
+<Callout variant="important">
+  Automatic log detection is not available for the Lite browser agent.
+</Callout>
+
+## Enable automatic log collection (For Existing Browser Apps) [#enable-automatic-log]
+
+To enable automatic log collection for an existing browser app:
+
+<Steps>
+
+  <Step>
+    Go to **[one.newrelic.com](https://one.newrelic.com) > All Capabilities > Browser**.
+  </Step>
+
+  <Step>
+    Select your browser app.
+  </Step>
+
+  <Step>
+    In the left-hand menu, click **Application settings**.
+  </Step>
+
+  <Step>
+    Ensure the **Pro** or **Pro + SPA** browser agent is selected.
+  </Step>
+
+  <Step>
+    Toggle **Browser Logs** to **ON**.
+  </Step>
+
+  <Step>
+    Toggle the required verbosity level to **ON**, you can choose from:
+    
+      * **Error**
+      * **Warn**
+      * **Info**
+      * **Debug**
```

Comment:
```suggestion
      * **Debug**
      * **Trace**
```

Reply:
This section is missing one of the log levels

Reply:
It may also be worth noting that the log levels here are inclusive of lesser verbosities